Header header

string receiver_name  # 接收车辆名称
float64 x  # 目标点X坐标
float64 y  # 目标点Y坐标
float64 yaw # heading
bool if_straight #判断接下来是否走直线
